### ISSUE TITLE
Request Body for GET requests dont display in path /docs

### ISSUE THREAD

**Issue by AndrejCoder** [tiangolo/fastapi#1196]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description
Request Body for GET requests dont display in page path /docs. Is it bug or feature?

![request_body_get](https://user-images.githubusercontent.com/12463504/78032886-5d788e00-736e-11ea-9af6-52075f10735e.png)

**Comment by nsidnev** [comment]
Have you looked [here](https://github.com/tiangolo/fastapi/issues/713)?

**Comment by Andrjuxa** [comment]
Yes, I understand that this has not yet been released, because the lines have not been changed ?

METHODS_WITH_BODY = set(("POST", "PUT", "DELETE", "PATCH"))

**Comment by nsidnev** [comment]
Yes, it seems so. Adding `"GET"` to this set will show the request body in the swagger docs. But I think this is still not very useful for Swagger (and ReDoc, which just won't show this route).

As I understand it, a Swagger can't send a request on this route. More information can be found [here](https://stackoverflow.com/questions/48749252/swagger-typeerror-failed-to-execute-fetch-on-window-request-with-get-head) and in Swagger-UI repo.

**Comment by tiangolo** [comment]
Yep, what @nsidnev said :point_up: 

If possible, to avoid headaches with interoperability with other tools, you should use `POST` instead if you want to receive a body.

**Comment by chris-allnutt** [comment]
Open API v3.0 spec does not support request body for GET requests
Relevant documentation here https://swagger.io/docs/specification/describing-request-body/

> GET, DELETE and HEAD are no longer allowed to have request body because it does not have defined semantics as per RFC 7231

You can still override this if you really want it, but I'm not sure if the doc generation will process it https://fastapi.tiangolo.com/advanced/extending-openapi/
